### ISSUE TITLE
make output colorful (good looking) in lscorecard.py.

### ISSUE THREAD

**Issue by sipah00** [sipah00/CrickFev#8]
the output of lscorecard command should be colorful (good looking) as done in lscore.py.

**Comment by aditodkar** [comment]
Can I work on this ? Can you please explain exactly how should be the color scheme.

**Comment by sipah00** [comment]
@aditodkar, yes, you can work and color scheme will be good as in lscore.py and make it better if you have good color scheme idea.
